tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/EmailRecoveryActionSetMethod.scala

```diff
@@ -56,16 +57,23 @@ class EmailRecoveryActionMethodModule extends AbstractModule {
 object EmailRecoveryActionConfiguration {
 
   val DEFAULT_MAX_EMAIL_RECOVERY_PER_REQUEST: Long = 5
+  val DEFAULT_USER_HORIZON: Duration = DurationParser.parse("15", ChronoUnit.DAYS)
 
   def from(propertiesProvider: PropertiesProvider): EmailRecoveryActionConfiguration = {
     val maxEmailRecoveryPerRequest: Option[Long] = Try(propertiesProvider.getConfiguration("jmap"))
       .map(configuration => configuration.getLong("emailRecoveryAction.maxEmailRecoveryPerRequest"))
       .toOption
-    EmailRecoveryActionConfiguration(maxEmailRecoveryPerRequest = maxEmailRecoveryPerRequest.getOrElse(DEFAULT_MAX_EMAIL_RECOVERY_PER_REQUEST))
+    val userHorizon: Option[Duration] = Try(propertiesProvider.getConfiguration("jmap"))
+      .map(configuration => configuration.getDuration("emailRecoveryAction.userHorizon"))
```

Comment:
```suggestion
      .map(configuration => configuration.getDuration("emailRecoveryAction.restorationHorizon"))
```

user part is useless: in JMAP this is necessrily a user.

I found it useful to specify wich was the horizon...